database/migrations/20200102172244-create-ticket.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable("Tickets", {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4
+      },
+      ticket_no: {
+        type: Sequelize.INTEGER,
+        allowNull: false
+      },
```

Comment:
Ticket no here should be uuid. We need a unique hash looking string. Maybe just a string that has the event name and user details somewhere.

Reply:
I was following the db diagram, I'd change it to String